src/components/cover-sheet/CoverSheet.tsx

```diff
@@ -31,6 +32,7 @@ export const CoverSheet = ({
   label,
   onClose,
   open,
+  showCloseButton = true,
```

Comment:
What do you think about making this boolean prop default to `false`? I probably didn't follow this rule in the past, but I have come to see that it is more idiomatic to have boolean props default to false, so when you want to change them you can use the shorthand of just the prop name.

```
<
  hideCloseButton
/>
```

```
< 
  showCloseButton={false}
/>
```